analytical_engine/core/app/pregel/pregel_compute_context.h

```diff
@@ -59,6 +60,7 @@ class PregelComputeContext {
 
     messages_in_.Init(inner_vertices, {});
     halted_.Init(inner_vertices, false);
+    vid_parser_.Init(frag.fnum(), 0);
```

Comment:
`0` might cause problem when doing gid -> lid. Could we use the actual original vertex label number there?

Reply:
in the context, we do not know the label_num, and the vid_parser_ only use to parse fid